api/pandas/time_series.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Time Series as pd.Series
+nav_order: 1
+parent: Pandas
+grand_parent: API
+---
+
+# Get Time Series as pandas Series
+
+## `datacommons_pandas.build_time_series(place, stat_var, measurement_method=None,observation_period=None, unit=None, scaling_factor=None)`
+
+Returns a `pandas.Series` mapping date to value for a `place` based on the
```

Comment:
not sure if "mapping" is used in pandas or somether Series specific words

Reply:
Changed this block to "
Returns a `pandas.Series` representing a time series for the [`place']https://datacommons.org/browser/Place] and
[`stat_var`](https://datacommons.org/browser/StatisticalVariable) satisfying any optional parameters.
"